apps/opik-backend/src/test/java/com/comet/opik/infrastructure/bi/BiEventListenerTest.java

```diff
@@ -155,6 +156,8 @@ void shouldReportFirstTraceCreatedEvent(UsageReportService usageReportService) {
 
         Trace trace = factory.manufacturePojo(Trace.class);
 
+        cacheManager.evict(Metadata.FIRST_TRACE_CREATED.getValue(), false).block();
```

Comment:
flaky test https://github.com/comet-ml/opik/pull/2058/checks?check_run_id=41816652689

Reply:
Not sure why it didn't link but it's here: https://github.com/comet-ml/opik/actions/runs/14889186341